examples/deprecation/deprecation.bal

```diff
@@ -0,0 +1,68 @@
+// The constant `CITY` is marked as deprecated using the `@deprecated` annotation and the markdown
+// documentation `# # Deprecated`.
+# Define a constant to represent the city.
```

Comment:
```suggestion
# A city code for Colombo city.
```

Reply:
Shall we go with "Define... " since that's what we've done in general?

Reply:
Not suitable for API docs right? For the usual doc comments for BBEs, that's fine

Reply:
Aargh, missed that it was docs. Sorry. +1 to change. 

Shall we say "The city code.." instead of "A city code..."?